src/pages/preferences/preferencesTabs/cliTab/DownloadCard.tsx

```diff
@@ -28,76 +27,76 @@ export const DownloadCard = () => {
-            <StyledLink href={cliDocumentationUrl}>documentation</StyledLink> or
-            run &nbsp;{" "}
-          </Body>
-          <InlinePre>evergreen --help or evergreen [command] --help</InlinePre>{" "}
-          <Body>for additional assistance.</Body>
-        </CardDescription>
-        <CardGroup>
-          {topBinaries.map((binary) => (
-            <CliDownloadBox
-              key={`downloadBox_${binary.url}`}
-              title={
-                prettyDisplayNameTop[binary.displayName] || binary.displayName
-              }
-              link={binary.url}
-            />
-          ))}
-        </CardGroup>
-        <Accordion
-          title={<StyledLink>Show More</StyledLink>}
-          toggledTitle={<StyledLink>Show Less</StyledLink>}
-          contents={<ExpandableLinkContents clientBinaries={otherBinaries} />}
-          toggleFromBottom
-          showCaret={false}
-        />
-      </Container>
-    </>
+    <Container>
+      <Subtitle>Command-Line Client</Subtitle>
+      <CardDescription>
+        <Body>
+          View the{" "}
+          <StyledLink href={cliDocumentationUrl}>documentation</StyledLink> or
+          run &nbsp;{" "}
+        </Body>
+        <InlinePre>evergreen --help or evergreen [command] --help</InlinePre>{" "}
+        <Body>for additional assistance.</Body>
+      </CardDescription>
+      <CardGroup>
+        {topBinaries.map((binary) => (
+          <CliDownloadBox
+            key={`downloadBox_${binary.url}`}
+            title={
+              prettyDisplayNameTop[binary.displayName] || binary.displayName
+            }
+            link={binary.url}
+            description={descriptions[binary.displayName]}
+          />
+        ))}
+      </CardGroup>
+      <Accordion
+        title="Show More"
+        toggledTitle="Show Less"
+        contents={<ExpandableLinkContents clientBinaries={otherBinaries} />}
+        showCaret={false}
+      />
+    </Container>
   );
 };
 
 interface CliDownloadBoxProps {
   title: string;
   link: string | null;
+  description?: string;
 }
-const CliDownloadBox: React.FC<CliDownloadBoxProps> = ({ title, link }) => {
+const CliDownloadBox: React.FC<CliDownloadBoxProps> = ({
+  title,
+  description,
+  link,
+}) => {
   const { sendEvent } = usePreferencesAnalytics();
   return (
-    <>
-      {/* @ts-expect-error */}
-      <CliDownloadCard>
-        {/* @ts-expect-error */}
-        <CliDownloadTitle>{title}</CliDownloadTitle>
-        <CliDownloadButton
-          onClick={() => {
-            sendEvent({
-              name: "CLI Download Link",
-              downloadName: title,
-            });
-          }}
-          href={link}
-          disabled={!link} // @ts-expect-error
-          as="a"
-        >
-          Download
-        </CliDownloadButton>
-      </CliDownloadCard>
-    </>
+    <CliDownloadCard>
+      <CliDownloadTitle>{title}</CliDownloadTitle>
+      {description && <span>{description}</span>}
```

Comment:
Thoughts on using MongoDB's `Description` tag here? Kind of gives it a nice gray contrast 🙂

<img width="875" alt="Screen Shot 2022-01-27 at 12 23 31 PM" src="https://user-images.githubusercontent.com/47064971/151411660-05fcc4b7-af33-44fb-ba4e-1b4a710f1bcd.png">

Reply:
SGTM done!